app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -1266,6 +1309,20 @@ class VaultAddEditViewModel @Inject constructor(
         }
     }
 
+    private fun handleFido2RegisterCredentialResultReceive(
+        action: VaultAddEditAction.Internal.Fido2RegisterCredentialResultReceive,
+    ) {
+        when (val result = action.result) {
+            is Fido2CreateCredentialResult.Error -> {
+            }
```

Comment:
💭  just to check, should the error be ignored?

Reply:
It should not. I was in the middle of moving this result handling to `Fido2CompletionManager` when we migrated repositories so I had to commit real quick and transfer these changes. It's been updated appropriately.